### PR TITLE
Add Google analytics

### PR DESCRIPTION
**Pull request by assertchris**
Please comment with your google accounts so I can add them.

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -12,6 +12,17 @@
 
     <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+
+    @if (env('APP_ENV') === 'production')
```
Comment (m1guelpf) [inline resources/views/layouts/app.blade.php]: You shouldn't durectly use the ENV helper, when the configuration is cached it doesn't work. You should probably replace it with `app()->environment('production')`

**Comment by m1guelpf** [comment]
soy@miguelpiedrafita.com

**Comment by assertchris** [comment]
I'd suggest having the landing page extend the app layout (that's what the app layout is for) but that introduces a load of other questions; like whether we should move to tailwind for the home page.

**Comment by m1guelpf** [comment]
@assertchris I'd leave the landing as-is, as for example the landing doesn't need to load any scripts. Also, I'm in favour of moving to Tailwind, but maybe we should discuss that in the Design issue...

**Comment by assertchris** [comment]
@m1guelpf yeah, I guess not loading scripts makes landing faster. Ok, cool. I've added the analytics there too.

**Pull request merged**